examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -122,54 +241,27 @@ EmberAfStatus OTAProviderExample::HandleQueryImage(chip::app::CommandHandler * c
         MutableCharSpan uri(uriBuf, kUriMaxLen);
         chip::bdx::MakeURI(nodeId, CharSpan(mOTAFilePath, strlen(mOTAFilePath)), uri);
         ChipLogDetail(SoftwareUpdate, "Generated URI: %.*s", static_cast<int>(uri.size()), uri.data());
-    }
 
-    // Set Status for the Query Image Response
-    switch (mQueryImageBehavior)
```

Comment:
@carol-apple @selissia I removed the handling of `mQueryImageBehavior`, not sure if we still need this. If we don't want this, then I can remove the other occurrences of this.

Reply:
Keeping the QueryImage behaviour option, resolving.